skate-plugin/src/main/resources/messages/skateBundle.properties

```
@@ -0,0 +1,5 @@
+skate.configuration.title=Skate Settings
+skate.configuration.choosePath.title=Choose file:
+skate.configuration.choosePath.dialog.title=Choose Changelog File
+skate.configuration.enableWhatsNew.title=Enable "What's New" Panel
+skate.configuration.enableWhatsNew.description=Will show "What's New" Panel if checked
```

nit: let's not quote these. Also suggesting a tweak to the description

```suggestion
skate.configuration.enableWhatsNew.title=Enable What's New Panel
skate.configuration.enableWhatsNew.description=If enabled, shows the What's New Panel if there are new changes detected in a specified changelog file.
```